src/Stream.sol

```diff
@@ -66,6 +65,8 @@ contract Stream is IStream, Initializable, ReentrancyGuard {
      * ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
      */
 
+    uint256 public constant RATE_DECIMALS = 1e6;
```

Comment:
I would name this `RATE_DECIMALS_MULTIPLIER`
I would expect RATE_DECIMALS to be 6 not 10^6

Reply:
thank you, fixed!